### PR TITLE
more importer adjustments

### PR DESCRIPTION
**Pull request by escattone**
* add tags
* use "translation_of" rather than "translationof"
* ~~remove use of "rendered_html"~~
* remove query for translations

### CODE REVIEW

**Comment by peterbe** [comment]
Without rendered_html we won't be able to work on the builder. That means if we land this, we won't be able to work on it until we fully solve the KS rendererer.

**Comment by escattone** [comment]
> Without rendered_html we won't be able to work on the builder. That means if we land this, we won't be able to work on it until we fully solve the KS rendererer.

👍 I'll take that out of this PR and keep it for my own work.

File: content/scripts/importer.js [hunk 1]
```diff
@@ -381,18 +381,18 @@ async function processDocument(
   }
   if (doc.parent_slug) {
     assert(doc.parent_locale === "en-US");
-    meta.translationof = doc.parent_slug;
+    meta.translation_of = doc.parent_slug;
```
Comment (peterbe) [inline content/scripts/importer.js]: 👍

**Comment by escattone** [comment]
@peterbe I checked the production DB and `https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/Video` really doesn't have any tags at all.

**Comment by peterbe** [comment]
> @peterbe I checked the production DB and `https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/Video` really doesn't have any tags at all.

So it's not a bug in the dumper but a reality of the mysql data?

**Comment by escattone** [comment]
> > @peterbe I checked the production DB and `https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/Video` really doesn't have any tags at all.
> 
> So it's not a bug in the dumper but a reality of the mysql data?

Yes, that's correct. A separate question is what we should be doing in this case. In other words, might it be better if translated documents always inherit their parent's tags? Maybe not, but just wondering. @wbamberg @Elchi3 What do you think?

**Pull request merged**